docs/windows.md

```diff
@@ -243,7 +243,7 @@ $networkName = 'nat'
 # Get-HnsNetwork is available once you have enabled the 'Hyper-V Host Compute Service' feature
 # which must have been done at the Quick setup above
 # Enable-WindowsOptionalFeature -Online -FeatureName containers -All
-# Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V-All -All
+# Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All
```

Comment:
Does not seem related to this PR, can you open a new one for this fix?

Reply:
I've removed it. I'll open a new one to fix it.